docs/developer/decision-records/2022-09-18-ids-requests-filtering/README.md

```diff
@@ -0,0 +1,42 @@
+# Add filtering to IDS requests
```

Comment:
```suggestion
# Add filtering to IDS catalog requests
```
This is important as it limits the scope significantly.

Reply:
Done.